web/src/pages/orders/[orderId]/hooks.js

```diff
@@ -0,0 +1,38 @@
+import { useCategories } from "@/hooks/redux/useCategories";
+import { useOrder, useOrders } from "@/hooks/redux/useOrders";
+
+const useHooks = (orderId) => {
+  const { orders, isLoading: ordersLoading, error: ordersError } = useOrders();
+  const { order, isLoading: orderLoading, error: orderError } = useOrder(orderId);
+  const { categories, isLoading: categoriesLoading } = useCategories();
```

Comment:
i dont know why you still need to fetch the orders and categories
```suggestion
  const { order, isLoading: orderLoading, error: orderError } = useOrder(orderId);
```